simulations/getting-started.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Getting Started
+---
+
+import { Tabs, Tab } from 'nextra/components';
+
+This guide will walk you through the basic setup required to run your first simulation and see the results in LangWatch.
+
+For more in-depth information and advanced use cases, please refer to the official [`scenario` library documentation](https://github.com/langwatch/scenario).
+
+## 1. Installation
+
+First, you need to install the `scenario` library in your project. Choose your language below.
+
+<Tabs>
+  <Tab title="Python">
+    ```bash
+    pip install scenario-langwatch
+    ```
+  </Tab>
+  <Tab title="TypeScript">
+    ```bash
+    npm install @langwatch/scenario
+    ```
```

Comment:
pip -> uv
npm -> pnpm

wdty?

we are at the bleeding edge of technology, let's push people :P